examples/README.md

```diff
@@ -44,6 +44,23 @@ In addition to the above, there are additional examples showing how to use the c
 2. `xpub-from-xpriv` - allows you to generate an xPub key from an xPriv key. To you use it you just need to replace the `xPriv` variable with your own xPriv key.
 3. `generate-totp` - allows you to generate and check validity of a TOTP code for client xPriv and a contact's PKI
 
+## Webhook example and Use Cases
+
+The [webhook.ts](./webhook.ts) example demonstrates how to set up a webhook notifier with the spv-wallet-js-client library, utilizing Fastify as the server to handle incoming webhook events. This setup allows developers to handle real-time updates from the spv-wallet system, such as transaction notifications, status updates, and custom events. The example is designed to be adaptable; any server framework can be substituted to handle these notifications, as it follows a flexible handler interface.
+
+#### Registering Event Handlers
+
+In this setup, we can register handlers only for specific event types, and these events mirror those defined in the `spv-wallet`.
+To add a new event type for handling, it must first be implemented in `spv-wallet`, then mapped within the `Events` object in this client. This setup ensures that event handling in the client is synchronized with available events from the `spv-wallet`.
```

Comment:
If it's possible please link to the currently supported Event types as github permalink 🙏

Reply:
Best I can do is a link to types file :D

Reply:
sure, approved